qaframework-bdd-tests/src/test/java/org/openmrs/contrib/qaframework/automation/OpenConceptLabSteps.java

```diff
@@ -0,0 +1,127 @@
+    }
+    
+    @After(RunTest.HOOK.SELENIUM_OPEN_CONCEPT_LAB)
+    public void destroy(){
+        quit();
+    }
+    
+    @Given("a user clicks on configure Metadata link from home page")
+    public void launchMetadataDashboard() {
+    	configureMetadataPage = (ConfigureMetadataPage) homePage.goToConfigureMetadata();
+    }
+    
+    @When("the system loads the configure metadata page")
+    public void systemLoadsConfigureMetadataPage() {
+    	assertTrue(textExists("Manage OCL"));
+    }
+ 
+    @And("a user clicks Manage OCL link from configure metadata page")
+    public void loadOpenConceptLabPage() {
+        openconceptlabpage = (OpenConceptLabPage) configureMetadataPage.goToOpenConceptLabPage();
+    }
+    
+    @Then("the system loads Open Concept Lab page")
+    public void systemLoadsOpenConceptPage() {
+    	assertTrue(textExists("Setup Subscription"));
+    }
+    
+    @When("a user clicks on Setup subscription button")
+    public void loadSubscriptionPage() {
+      subscriptionPage = (SubscriptionPage) openconceptlabpage.clickOnsetupSubscriptionButton();
+    }
+ 
+    @Then("the system loads Subscription page")
+    public void systemLoadsSubscriptionPage() {
+        assertTrue(textExists("Show/hide advanced..."));
+    }
+
+    @And("a user enters the URL of a new released dictionary")
+    public void enterDictionaryUrl() {
+        subscriptionPage.enterSubscriptionURL(releasedDictionaryUrl);
+    }   
+
+    @And("a user enters the Token url")
+    public void enterTokenUrl(){
+         subscriptionPage.enterTokenURL(tokenUrl);
+    }
+
+    @And ("a user clicks on the Save Changes button")
+    public void saveChanges(){
+        openConceptLabSuccessPage = (OpenConceptLabSuccessPage) subscriptionPage.clickSaveChangesButton();
+    }
+
+    @Then("the system loads Open Concept Lab Success page")
+    public void systemLoadsOpenConceptLabSuccessPage(){
+    	 assertTrue(textExists("Import from subscription server"));  
+    }
+
+    @And("a user clicks import from Subscription server button")
+    public void LoadsOpenConceptLabPage(){
+        openConceptLabSuccessPage.clickOnImportFromSubscriptionButton();
+        openConceptLabSuccessPage.waitForPageToLoad();
+    }
+
+    @Then("the API should be displayed on the previous imports")
+    public void displayAmongPreviousImports(){
+         assertNotNull(openConceptLabSuccessPage.getpreviousImportsList());
+    } 
+
+    @When("a user clicks edit subscription button")
+    public void loadsSubscriptionPage(){
+         subscriptionPage = (SubscriptionPage) openConceptLabSuccessPage.clickEditSubscriptionButton();  
+     } 
+    
+    @And("a user edits the URL of a released dictionary")
+    public void editDictionaryUrl() {
+        subscriptionPage.enterSubscriptionURL("https://api.qa-refapp.openmrs.org/openmrs/dictionary/concept.htm?conceptId=162172");
+    }   
+
+    @And("a user edits the Token url")
+    public void editTokenUrl(){
```

Comment:
```suggestion
    public void editTokenUrl()  {
```
Check all these similar changes and correct them appropriately.